### PR TITLE
Fixed issue #2888: 'Show keypad' button floating around

### PR DESCRIPTION
**Pull request by Antrikshy**
This should fix issue #2888. The 'show keypad' button now sits where it should and matches the hint and scratchpad buttons. Software keyboard has been tweaked slightly. The buttons are now `simple-button`s instead of the weird `green_button` class (which had no CSS styling anywhere LOL).

screen shot 2015-01-26 at 11 48 43 pm

Things are now pixel-perfect!

I have also improved the coloring on the main green buttons that I edited with the recent overhaul. They match the orange buttons better.

> I had some trouble with my local clone and had to fiddle around a bit. Ignore the "changes" to `models.js` and `views.js`. Unless I'm missing something, the changes it shows for both those files match the current versions of those files in the develop branch. Nothing should change. Sorry if it causes any trouble.

### CODE REVIEW

**Comment by rtibbles** [comment]
Yeah, I think the green-button class was added pre-bootstrap and probably got culled during bootstrap integration. Will check the changes to models and views.

File: kalite/distributed/static/js/distributed/software-keyboard.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ window.SoftwareKeyboardView = Backbone.View.extend({
                 .appendTo(self.software_keyboard);
 
             jQuery.each(row, function(j, key) {
-                var keySpan = $("<div class='.col-xs-4'><button class='key green_button " + (key === "bs" ? "key-bs" : "") + "' value='" + key + "'>" + (key === "bs" ? "Del" : key) + "</button></div>").appendTo(rowDiv);
+                var keySpan = $("<div class='.col-xs-4'><button class='key simple-button " + (key === "bs" ? "key-bs" : "") + "' value='" + key + "'>" + (key === "bs" ? "Del" : key) + "</button></div>").appendTo(rowDiv);
```
Comment (rtibbles) [inline kalite/distributed/static/js/distributed/software-keyboard.js]: One thing here - the 'Del' key is slightly misaligned on the keypad. I think it is inheriting some styling that changes the font size and makes it misaligned, can you play around with it (first, just make its styling the same as all the other keys), and see if that fixes the alignment issue too?

**Comment by rtibbles** [comment]
Merges seem fine. You're just going to get blamed for code you didn't write now though :)

**Comment by Antrikshy** [comment]
Hah "blamed".

I totally forgot about that Del button. It's not pixel perfect, I guess. I can't seem to find why it is misaligned. It must be because of the way it's added using JavaScript. I'll look into it today. Maybe make it an empty button somewhere in the template and only modify it using JS.

**Comment by rtibbles** [comment]
In khan-lite.css there is a class called .key-bs that is being applied to the "Del" key. I think just deleting this definition (and then deleting its application in the line I annotated) should make this align properly.

**Comment by MCGallaspy** [comment]
I wanted to check this out, but I can't figure out where the show keypad button is supposed to be... :(

Steps to reproduce?

**Comment by Antrikshy** [comment]
@MCGallaspy I have fixed the show keypad button. The Hide Keypad button *is* the Show Keypad button. It changes contextually. The only thing left to do is to align the Del button correctly.

**Comment by aronasorman** [comment]
@Antrikshy or @rtibbles: @MCGallaspy can't seem to find the "Show Keypad" button, any guidance on this?

**Comment by rtibbles** [comment]
Looking at the basic addition exercise is the best test case.

**Comment by Antrikshy** [comment]
screen shot 2015-01-29 at 10 15 55 am

Everything is fixed and cleaned up. Say hello to keypad 2.0 (more like 1.0.1)!

**Comment by Antrikshy** [comment]
By the way, @rtibbles, when I click any of the buttons on the keypad, it says "There are still more parts of this question to answer." Is this a known issue or just something on my end because I am missing that assessmentitems file? Seems fishy to me.

**Comment by MCGallaspy** [comment]
Keypad looks beautiful, but I get the following error when I click on it:
```
 Uncaught TypeError: Cannot read property 'setRangeText' of undefined software-keyboard.js:58
```

**Comment by Antrikshy** [comment]
The keypad clicks are unrelated to the changes in this PR. I have the same problem with the develop branch. I have created a new issue #2907 for this and I'm looking into it. This PR is ready to be merged.

**Pull request merged**